anjali29/bio.html

```diff
@@ -0,0 +1,204 @@
+            </p>
+         </div>
+         <div>
+            <p>
+            <h2 style="text-decoration:underline">Co-curricular activities</h2>
+            <ol>
+               <li> Participated in poster rpresentation event at S.E.Spolytechnic,Solapur.</li>
+               <li> Participated in quiz competition event at S.E.Spolytechnic,Solapur</li>
+               <li>  Worked as a volunteer in “Orchitech2015”.</li>
+            </ol>
+            </p>
+         </div>
+         <div>
+             <h2 style=" text-decoration: underline;">Educational Qualifications</h2>
+            <table style="border: 1px dashed black;">
+                <tr style=" font-weight:bold;">
+                    <th colspan="2" style=" padding: 2px;
+                        border: 1px solid #E88741;">institution</th>
+                     <td style=" padding: 2px;
+                        border: 1px solid #E88741; ">year of passing</td>
+                    <td style=" padding: 2px;
+                        border: 1px solid #E88741;">certificate / degree </td>
+                </tr>
+                 <tr>
+                    <td style=" padding: 2px;
+                         border: 1px solid #E88741;"> <a href="https://www.orchidengg.ac.in/" target="_blank">NK ORCHID COLLAGE <td>
+                           
+                       <img src="images/2.jpg" alt="Image is not available" height="200px" width="200px"></a>
+                   </td>
+                     </td>
+                   <td style=" padding: 2px;
+                        border: 1px solid #E88741;">2017</td>
+                    <td style=" padding: 2px;
+                        border: 1px solid #E88741;">degreecertificate</td>
+                 </tr>
+                <tr>
+                    <td style=" padding: 2px;
+                        border: 1px solid #E88741;"> <a href="http://www.sespolytechnic.org.in/" target="_blank">SES POLYTECHNIC solapur<td>
+                       <img src="images/3.jpg" alt="Image is not available" height="200px" width="200px"></a>
+                   </td>
+                   </td>
+                   
+                  <td style=" padding: 2px;
+                        border: 1px solid #E88741;">2014</td>
+                    <td style=" padding: 2px;
+                        border: 1px solid #E88741;">degreecertificate</td>
+                </tr>
+             </table>
+         </div>
+         <div>
+            <h2 style="text-decoration:underline">Personality skills:</h2>
+            <p>Positive attitude, hard working ,good team player</p>
+            <div>
+               <h2 style="text-decoration:underline">Personal Information:</h2>
+               <p>
+               <ul>
+                  <li>
+                     <span style="font-weight:bold">Languages Known:</span>English,marathi,hindi
```

Comment:
explain you use p tag

Reply:
<p> tag is used to designate a paragraph.